models/comments.models.js

```diff
@@ -0,0 +1,13 @@
+const db = require("../db/connection");
+
+exports.removeComment = (comment_id) => {
+  return db.query(
+    `DELETE FROM comments
+    WHERE comment_id = $1`,
+    [comment_id]
+  ).then(({rowCount}) => {
+    if (rowCount === 0) {
+        return Promise.reject({status: 404, msg: "Comment not found"})
+    }
+  })
+};
```

Comment:
Spot on model - great use of the promise reject here to inform the client that a comment that wasn't found